challenges/compareSort/compare.test.js

```diff
@@ -0,0 +1,36 @@
+const { Movies } = require("./movies");
+const { sortTitle, sortYear } = require("./sort");
+
+describe("Sorters", () => {
+  it("can sort movies by year", () => {
+    const movies = sortYear(Movies);
+    expect(movies.map((m) => m.title)).toEqual([
+      "The Intouchables",
+      "Valkyrie",
+      "Stardust",
+      "Ratatouille",
```

Comment:
Switch these two around because they both came out the same year, so it putting Ratatouille first because "R" is before "S"